### PR TITLE
Review pagination

### PR DESCRIPTION
**Pull request by fazo96**
Review pagination implemented in Reviews I Wrote tab, Profile reviews and Reviews About Me

__Note__: these changes require chlu-api-query v0.3.0, which is already deployed on our infrastructure

### CODE REVIEW

File: src/components/Reviews/index.js [hunk 1]
```diff
@@ -17,8 +17,8 @@ const styles = {
 class Reviews extends Component {
 
   render() {
-    const { classes, reviews, loading, crawling } = this.props
-    const reviewList = reviews ? Object.values(reviews) : []
+    const { classes, reviews = [], loading, crawling = false, onLoadMoreReviews, canLoadMore } = this.props
+    const empty = reviews ? !reviews.length : true
```
Comment (JohnWeisz) [inline src/components/Reviews/index.js]: IMO, this is nicer to read:
```
const empty = !reviews || !reviews.length
```

File: src/containers/Reputation/index.js [hunk 2]
```diff
@@ -31,7 +38,9 @@ class Reputation extends Component {
 const mapStateToProps = state => ({
   reviews: state.data.reputation.reviews,
   loading: state.data.reputation.loading,
-  crawling: state.data.crawler.running
+  loadingPage: state.data.reputation.loadingPage,
+  canLoadMore: state.data.reputation.canLoadMore,
+  crawling: state.data.crawler.running,
```
Comment (JohnWeisz) [inline src/containers/Reputation/index.js]: trailing comma

File: src/containers/ReviewsIWrote/index.js [hunk 3]
```diff
@@ -11,19 +11,30 @@ class ReviewsIWrote extends Component {
...
+    if (!this.props.loading) this.props.readReviewsIWrote(false)
+  }
 
   render() {
-    const { reviews, editing, loading, error } = this.props
-
-    return <Reviews reviews={reviews} loading={loading} error={error} editing={editing} />
+    const { reviews, loading, loadingPage, canLoadMore, error } = this.props
+
+    return <Reviews
```
Comment (JohnWeisz) [inline src/containers/ReviewsIWrote/index.js]: We discussed this quite some time ago now, but we generally agreed to use this syntax for multi-line components returned:
```
return (
  <elem>
    ...
  </elem>
)
```

File: src/store/modules/data/reputation.js [hunk 4]
```diff
@@ -9,12 +9,17 @@ import { DELETE_WALLET } from './wallet';
 const READ_REPUTATION_LOADING = 'reviewsaboutvendor/READ_REPUTATION_LOADING'
 const READ_REPUTATION_SUCCESS = 'reviewsaboutvendor/READ_REPUTATION_SUCCESS'
 const READ_REPUTATION_ERROR = 'reviewsaboutvendor/READ_REPUTATION_ERROR'
+const itemsPerPage = 5
```
Comment (JohnWeisz) [inline src/store/modules/data/reputation.js]: I would stick to conventions and use UPPER_CASE naming here.

**Comment by fazo96** [comment]
Syntax issues addressed on master

**Pull request merged**